src/main/java/io/knotx/junit5/util/RequestUtil.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2018 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.junit5.util;
+
+import io.reactivex.Single;
+import io.reactivex.functions.Consumer;
+import io.vertx.junit5.VertxTestContext;
+
+public interface RequestUtil {
```

Comment:
Using java `interface` to make util class is not a good practice. There is wide discussion about that you may find in the Internet. Please change it to `final class`.